config/default.yml

```
@@ -165,8 +165,14 @@ Rails/DynamicFindBy:
   StyleGuide: 'https://rails.rubystyle.guide#find_by'
   Enabled: true
   VersionAdded: '0.44'
+  VersionChanged: '2.5.3'
```

The next version hasn't been decided yet, but will probably be 2.6 and `VersionXxx` does not always require a patch version.

```suggestion
  VersionChanged: '2.6'
```